package.json

```diff
@@ -54,9 +54,11 @@
   },
   "homepage": "http://ftrack.com",
   "dependencies": {
+    "isomorphic-ws": "^5.0.0",
     "loglevel": "^1.8.1",
     "moment": "^2.29.4",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "ws": "^8.13.0"
```

Comment:
i think we want to add ws as a peerDependency to not require it being installed when used for browser purposes

Reply:
Yes. Done